### PR TITLE
fix issue 4282

### LINKED ISSUE #4282: a bug about function formatLog
Please answer these questions before submitting your issue. Thanks!

1. What version of Go and beego are you using (`bee version`)?
go1.15.2 beego v1.12.2

2. What operating system and processor architecture are you using (`go env`)?
GOHOSTARCH="amd64"
GOHOSTOS="linux"

3. What did you do?
If possible, provide a recipe for reproducing the error.
A complete runnable program is good.
call logs.Debug("%d%%", 10)

4. What did you expect to see?
call logs.Debug("%d%%", 10)
print 10%

5. What did you see instead?
10% %!v(MISSING)

This is my modification:
image

### PR DESCRIPTION
**Pull request by AllenX2018**
close #4282

### CODE REVIEW

File: core/logs/log.go [hunk 1]
```diff
@@ -764,9 +764,7 @@ func formatLog(f interface{}, v ...interface{}) string {
 		if len(v) == 0 {
 			return msg
 		}
-		if strings.Contains(msg, "%") && !strings.Contains(msg, "%%") {
-			// format string
-		} else {
+		if !strings.Contains(msg, "%") {
```
Comment (flycash) [inline core/logs/log.go:767]: I think it doesn't resolve the problem.
```
formatLog("%d%%, ", 12, "hello") // expected `12%, hello`
```
does it misses "hello"?

Comment (AllenX2018) [reply core/logs/log.go:767]: The syntax is incorrect. Two placeholders are required for two parameters. You only provide one.
```go
Calling fmt.Sprintf("%d%%", 10,"hello") will not solve this problem.
```

Comment (flycash) [reply core/logs/log.go:767]: @AllenX2018 Sorry, I write a test:
```go
func Test_format(t *testing.T) {
	res := formatLog("%d%%", 10)
	assert.Equal(t, "10%", res)

	res = formatLog("%d", 12)
	assert.Equal(t, "12", res)

	res = formatLog("%%", 12)
	assert.Equal(t, "% 12", 12)
}
```

And it failed.

Comment (AllenX2018) [reply core/logs/log.go:767]:
```go
res = formatLog("%%%d", 12)
assert.Equal(t, "%12", 12)
```

**Pull request closed without merging**